templates/admin/adherent/_summary.html.twig

```diff
@@ -2,6 +2,10 @@
     <a href="{{ path('admin_app_adherent_edit', { id: adherent.id }) }}">
         {{ adherent.fullName }}
     </a>
+{% elseif is_granted('ROLE_APP_ADMIN_ADHERENT_SHOW') %}
```

Comment:
It seams that `ROLE_APP_ADMIN_ADHERENT_SHOW` role does not exist and it should be `ROLE_APP_ADMIN_ADHERENT_VIEW` instead🤔

Reply:
wow indeed, good catch